src/app/pages/MediaArticlePage/PagePromoSections/LastestMediaSection/LatestMediaTypes/index.tsx

```diff
@@ -0,0 +1,35 @@
+export type LatestMedia = {
+  description: string;
+  duration: string;
+  firstPublished: string;
+  id: string;
+  imageUrl: string;
+  link: string;
+  title: string;
+  type: 'video' | 'audio';
```

Comment:
Have we defined this as a type somewhere (perhaps in globals types) and we could reuse? If not, is it worth creating it?

Reply:
I guess this depends if we're looking to use this data structure elsewhere in the application. From what I know this is the only place it's going to be used, so not sure if it's worth making it a global type and just keeping it co-located?

Reply:
Yeah that's fair - why don't we just add a new local type in this file and reference it here? We can always extract / refactor / reuse it later. Thanks.

Reply:
I was actually refering to the `type: 'video' | 'audio';` line (the GitHub conversations UI is a bit confusing - I prefer the files view!)

```
export type Media = 'video' | 'audio';
```

And then where it's declared
```
...
type: Media;
```

Reply:
Ahh yeah, makes sense, I've just added it!